altair/api.py

```diff
@@ -124,7 +124,7 @@ class SortItems(BaseObject):
 
     name = T.Unicode(default_value=None, allow_none=True)
     aggregate = T.Enum(['avg', 'sum', 'min', 'max', 'count'],
```

Comment:
Yes, once we get a basic implementation done we will probably start to pull
from the spec in certain places. We already have the spec bundled as a
Python dict in the Altair repo, so we can easily do this.

On Wed, Oct 14, 2015 at 11:14 PM, Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In altair/api.py
> <https://github.com/ellisonbg/altair/pull/70#discussion_r42079754>:
>
> > @@ -124,7 +124,7 @@ class SortItems(BaseObject):
> >
> >      name = T.Unicode(default_value=None, allow_none=True)
> >      aggregate = T.Enum(['avg', 'sum', 'min', 'max', 'count'],
>
> FYI, in vega-lite 0.8, avg will be changed to mean for consistency with
> Datalib. We will also support a bunch of other Vega/Datalib. aggregation
> operator.
>
> Vega-lite also include a JSON-schema
> <https://github.com/vega/vega-lite/blob/gh-pages/spec.json> in our
> releases <https://github.com/vega/vega-lite/releases>. You might want to
> consider referring to the JSON-schema rather than hard-coding supported
> enum. That said, hard-coding maybe better if you decide to support only a
> subset of values.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ellisonbg/altair/pull/70/files#r42079754>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com